OpenEXR/IlmImf/ImfDwaCompressor.cpp

```diff
@@ -2392,15 +2392,15 @@ DwaCompressor::uncompress
                             "(truncated file).");
     }
 
-    if (unknownUncompressedSize < 0  || 
-        unknownCompressedSize < 0    ||
-        acCompressedSize < 0         || 
-        dcCompressedSize < 0         ||
-        rleCompressedSize < 0        || 
-        rleUncompressedSize < 0      ||
-        rleRawSize < 0               ||  
-        totalAcUncompressedCount < 0 || 
-        totalDcUncompressedCount < 0) 
+    if ((SInt64)unknownUncompressedSize < 0  ||
+        (SInt64)unknownCompressedSize < 0    ||
+        (SInt64)acCompressedSize < 0         ||
+        (SInt64)dcCompressedSize < 0         ||
+        (SInt64)rleCompressedSize < 0        ||
+        (SInt64)rleUncompressedSize < 0      ||
+        (SInt64)rleRawSize < 0               ||
+        (SInt64)totalAcUncompressedCount < 0 ||
+        (SInt64)totalDcUncompressedCount < 0)
```

Comment:
If we don't want to add SInt64, I think most of these values can be sanity checked with  initalizeBuffers()